.wip/api-sketches/src/2-mcp-entrypoint-basic.ts

```diff
@@ -0,0 +1,97 @@
+import { Hono } from 'hono'
+import { basicAuth } from 'hono/basic-auth'
+
+import { WorkerEntrypoint, env } from 'cloudflare:workers'
+import { WorkersMCP, mcp } from 'dream:code'
+
+/**
+ * =============================================
+ * Stateless MCP server, implemented RPC-style
+ * with decorator-based documentation (no build step)
```

Comment:
i get it now. i didn't follow the use of parameters / decorator factory pattern here. my brain was stuck on vanilla decorator without arguments. this is brilliant